src/js/common/schemaform/review/ReviewCollapsibleChapter.jsx

```diff
@@ -151,7 +151,7 @@ export default class ReviewCollapsibleChapter extends React.Component {
                 {pageSchema &&
                   <SchemaForm
                     name={page.pageKey}
-                    title={page.reviewTitle || page.title}
+                    title={page.title.replace('Review ', '')}
```

Comment:
`reviewTitles` have been provided to omit the word "review" from chapter and page titles that include it, this will obviate that.

Reply:
This change seems very specific to the way the 526 form is using "Review," which might not be the same on other forms. Are there other ways we could handle this without disallowing the word review in any review page title?

Reply:
Good point Jeff, I meant to double check that reviewTitle was only being used to omit the word review, and it turns out it is being used in a variety of ways, so you're correct, this is far too narrow. Instead, title should accept a function as you proposed earlier.